src/main/pages/che-6/user-guide/version-control.adoc

```diff
@@ -128,6 +128,60 @@ endif::[]
 
 Once OAuth is configured, SSH keys are generated and uploaded automatically to GitHub by a user in the IDE in *Profile > Preferences > SSH > VCS* by clicking the _Octocat_ icon. You can connect to your GitHub account and choose repositories to clone, rather than having to manually type (or paste) GitHub project URLs.
 
+[id="configuring-github-oauth-for-openshift-infrastructure"]
+== Configuring GitHub oAuth for OpenShift infrastructure
+
+.Prerequisites
+
+* A deployed instance of Eclipse Che
+
+.Procedure
+
+OAuth for GitHub allows users to import projects using the SSH addresses (`git@`), push to repositories, and use the pull request panel. The steps listed in this section are performed by an administrative user.
```

Comment:
pull request panel -- is it a GUI element?

Reply:
Yes, CHE6 has the UI _Pull Request panel_

Reply:
Thanks, @artaleks9 !